SQL_Ruby_project/spending_methods.rb

```diff
@@ -0,0 +1,149 @@
+CENTS = 1
+ONE = 1
+TWO = 2
+INCLUDED = [[1]]
+CONVERSION = 100
+NEGATIVE = -1
+START = 0
+
+def start_database
+	db = SQLite3::Database.new("spending.db")
+	create_users_table = <<-SQL
+	  CREATE TABLE IF NOT EXISTS users(
+	    id INTEGER PRIMARY KEY,
+	    username VARCHAR(255),
+	    current_balance INT
+	  )
+	SQL
+	create_transactions_table = <<-SQL
+		CREATE TABLE IF NOT EXISTS transactions(
+			id INTEGER PRIMARY KEY,
+			time_stamp VARCHAR(255),
+			amount INT,
+			category VARCHAR(255),
+			user_id INT,
+			FOREIGN KEY (user_id) REFERENCES users(id)
+			)
+	SQL
+	db.execute(create_users_table)
+	db.execute(create_transactions_table)
+	db
+end
+
+def existing_name_checker(db, name)
+	if db.execute("SELECT 1 FROM users WHERE username=?", [name]) == INCLUDED

```

Comment:
you could refactor this to

just `db.execute("SELECT 1 FROM users WHERE username=?", [name]).present`